samples/nrf9160/aws_fota/Kconfig

```diff
@@ -44,6 +44,12 @@ config CLOUD_CLIENT_ID
 	default "your_client_id"
 endif
 
+if !USE_CLOUD_CLIENT_ID
+config CLOUD_CLIENT_ID_PREFIX
```

Comment:
Are both `CLOUD_CLIENT_ID_PREFIX` and `NRF_CLOUD_CLIENT_ID_PREFIX` needed ?
and what are the difference between those ?

They have exactly the same string prompt, and no help text.

Reply:
Yes, however `CLOUD_CLIENT_ID_PREFIX` could be wrapped in an additional `!USE_NRF_CLOUD` condition, because it only is used if `!USE_NRF_CLOUD && !USE_CLOUD_CLIENT_ID`.

I will improve the help text.